core/state/trie_prefetcher.go

```diff
@@ -308,7 +346,10 @@ func (sf *subfetcher) abort() {
 func (sf *subfetcher) loop() {
```

Comment:
not sure about this loop? do we need to record it?

Reply:
no , it does not affect main process cost